extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -178,6 +179,12 @@ export class AmpAnalytics extends AMP.BaseElement {
 
   /** @override */
   unlayoutCallback() {
```

Comment:
Ignore the following 6 lines. They're just a copy of #11713, and will be rebased away when that one merges.